roles/accprovision/manifests/net-operator-controller-deployment.yaml

```diff
@@ -0,0 +1,410 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: net-operator-controller
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: net-operator-controller
+  namespace: net-operator-controller
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    network-plugin: calico
+  name: net-operator-controller
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - '*'
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
+  - clusterrolebindings
+  verbs:
+  - '*'
+- apiGroups:
+  - ''
+  resources:
+  - nodes
+  - namespaces
+  - configmaps
+  - secrets
+  - pods
+  - services
+  - serviceaccounts
+  - serviceaccounts/token
+  - endpoints
+  - events
+  verbs:
+  - '*'
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - networkpolicies
+  verbs:
+  - list
+  - watch
+  - get
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - replicasets
+  - daemonsets
+  - statefulsets
+  verbs:
+  - '*'
+- apiGroups:
+  - aci.ctrl
+  resources:
+  - accprovisioninputs
```

Comment:
Let's replace all occurrences of "accprovision" here and below with "clusterinput" or "ClusterInput" or "cluster-input" as relevant.